app/models/team.rb

```diff
@@ -31,6 +31,10 @@ class Team < ActiveRecord::Base
   before_create :set_number
   before_save :set_last_checked, if: :checked
 
+  scope :without_recent_log_update, -> {
+    where.not(id: Activity.where(kind: 'status_update').where("created_at > ?", 26.hours.ago).pluck(:team_id))
```

Comment:
I dislike plucking the id's here but realistically speaking these will not be many - it's limited by the number of teams currently participating.